lib/liftoff/cli.rb

```diff
@@ -35,8 +35,18 @@ def global_options
           @options[:strict_prompts] = strict_prompts
         end
 
+        opts.on('--dependency-managers [NAME(s)]', 'Comma separated list of dependency managers to enable. Available options: cocoapods') do |list|
+          @options[:dependency_managers] = (list || "").split(",")
+        end
+
         opts.on('--[no-]cocoapods', 'Enable/Disable Cocoapods') do |use_cocoapods|
```

Comment:
I wasn't sure if I should remove this option or not. It wouldn't break people's `liftoffrc` but if there were a script that passes in options this needs to be here. Thoughts?